src/mozanalysis/metrics/__init__.py

```diff
@@ -83,7 +113,13 @@ def experiments_column_expr(self):
         else:
             raise ValueError
 
-    def build_query(self, metric_list, time_limits, experiment_slug):
+    def build_query(
+            self,
```

Comment:
I think our indentation style is to dedent the list of arguments once, and to dedent the `):` twice